packages/web-components/src/components/va-button/va-button.scss

```diff
@@ -48,11 +48,11 @@
 
   button:hover,
   button:focus {
-    background-color: var(--color-primary-darker);
+    background-color: var(--color-primary-dark);
   }
 
   button:active {
-    background-color: var(--color-primary-darkest);
+    background-color: var(--color-primary-darker);
```

Comment:
Shifts up a level due to names and values being changed per [this comment](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2228#issuecomment-1836918069)